Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1001,6 +1001,25 @@ Meanwhile `GKE internal TCP/UDP load balancer
 <https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#lb_source_ranges>`__
 does not, so the feature must be kept enabled in order to restrict the access.
 
+Service Proxy Name Configuration
+********************************
+
+Like kube-proxy, Cilium also honors ``service.kubernetes.io/service-proxy-name`` label and only
+manages services that contain a matching service-proxy-name label. This name can be configured
+by setting ``global.k8s.serviceProxyName`` option and the behavior is identical to that of
+kube-proxy. The service proxy name defaults to an empty string which instructs Cilium to
+only manage services not having ``service.kubernetes.io/service-proxy-name`` label.
+
+For more details on the usage of ``service.kubernetes.io/service-proxy-name`` label and its
+working, take a look at `this KEP
+<https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0031-20181017-kube-proxy-services-optional.md>`__
+from the Kubernetes team.
+
+.. note::
+
+    When setting the value for service proxy name make sure that default Kubernetes services
```

Comment:
Why managing `kube-dns` and `kubernetes` svcs matters? As long as other svc proxy is able to translate those two, cilium should be fine.

Reply:
I thought of the case when Cilium is deployed in a cluster without kube-proxy with a different service proxy name. But doing some more thinking, I think we can remove this as for kube-proxy replacement mode we consider using Cilium in completely compatible mode(empty service-proxy-name, like kube-proxy).

Reply:
I see. Maybe then elaborate a bit saying "if Cilium is meant to manage all services" or something similar along these lines.